lib/membrane_element_rtp_aac/payloader.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Membrane.RTP.AAC.Payloader do
+  @moduledoc "TODO"
+
+  use Membrane.Filter
+
+  alias Membrane.Buffer
+  alias Membrane.{AAC, RTP}
+  alias Membrane.RTP.AAC.Utils
+
+  def_input_pad :input, accepted_format: %AAC{encapsulation: :none}
+  def_output_pad :output, accepted_format: %RTP{}
+
+  def_options bitrate: [
+                spec: :lbr | :hbr
+              ],
+              frames_per_packet: [
+                spec: pos_integer()
+              ]
+
+  @impl true
+  def handle_init(_ctx, options) do
+    state =
+      options
+      |> Map.from_struct()
+      |> Map.put(:acc, [])
+
+    {_actions = [], state}
+  end
+
+  @impl true
+  def handle_stream_format(:input, _stream_fmt, _ctx, state) do
+    {[stream_format: {:output, %RTP{}}], state}
+  end
+
+  @impl true
+  def handle_demand(:output, size, :buffers, _ctx, state) do
+    {[demand: {:input, size}], state}
+  end
+
+  @impl true
+  def handle_buffer(:input, buffer, _ctx, state) do
+    @type buffer.payload :: binary()
```

Comment:
Is this `@type` annotation supposed to be there? And do we really need it?

Reply:
Breaking news from this morning is that it doesn't even work lol (gets treated as fn return value)